design/06_user_stories.md

```diff
@@ -0,0 +1,31 @@
+# Proposal number 6 : User stories
+
+In order to define the features required to the backup tool, we need user stories that cover use cases for the tool. These should be the point of focus of testing and development. They will represent **what** the code does, in order to create the **how** as a consequence. This proposal attempts to describe them.
+
+One may find this file to be incomplete, as it only represents the feature of a Minimum Viable Product. A backup over the network is not included, as well as an automated way to backup data, or a description of differences between backup. Those may be included in further versions.
+
+## Global configuration
+
+As a user **using the CLI**, I want to be able to:
+
+- `(1)` after plugging in my USB stick, HDD or SSD, create a new entry for it by providing all relevant information (name, type, location).
+- `(2)` edit and delete the properties of any support that has previously been registered.
+
+## Project configuration
+
+`(3)` As a user, I want to be able to initialize or edit a backup strategy for a specific repository by providing all the required configuration of that strategy : physical replication requirements, number of replication requirements, amount of security of said replications. For this repository I can choose to provide with a whitelist and/or a blacklist of security levels for each file/folder that should be included in the backup. The state of the backup should follow the following rules:
```

Comment:
To me individual security level might be omited in the mvp. A single security level for the project shall be enough. If we don't even have network locations at first, it should be ok